src/utils.h

```diff
@@ -2,13 +2,18 @@
 #ifndef DUNST_UTILS_H
 #define DUNST_UTILS_H
 
+#include <stdlib.h>
+
 /* replace all occurences of the character needle with the character replacement in haystack */
 char *string_replace_char(char needle, char replacement, char *haystack);
 
 /* replace all occurrences of needle with replacement in haystack */
 char *string_replace_all(const char *needle, const char *replacement,
                          char *haystack);
 
+/* Cut the given string at maxlen */
+char* string_cut_maxlen(char* string, size_t maxlen);
```

Comment:
Small nitpick, but I think `string_truncate` would be a better name for this function.